frontend/app/components/navbar/navbar.js

```diff
@@ -0,0 +1,35 @@
+'use client';
+import { Image, Nav } from 'react-bootstrap';
+import { Navbar } from 'react-bootstrap';
+import Container from 'react-bootstrap/Container';
+
+function Navibar() {
+    let currentPage = window.location.pathname;
+    return (
+        <Navbar style={{ overflow: 'auto' }}>
```

Comment:
what's this overflow do? i woulda thought auto is the default overflow

Reply:
erm aktually sir, that line has been deleted since i made it a hamborgir menu